core/src/software/aws/toolkits/core/ToolkitClientManager.kt

```diff
@@ -10,12 +10,12 @@ import software.amazon.awssdk.core.client.config.SdkAdvancedClientOption
 import software.amazon.awssdk.http.SdkHttpClient
 import software.amazon.awssdk.regions.Region
 import software.amazon.awssdk.services.s3.S3ClientBuilder
-import software.amazon.awssdk.services.s3.handlers.CreateBucketInterceptor
-import software.amazon.awssdk.services.s3.handlers.DecodeUrlEncodedResponseInterceptor
-import software.amazon.awssdk.services.s3.handlers.DisableDoubleUrlEncodingInterceptor
-import software.amazon.awssdk.services.s3.handlers.EnableChunkedEncodingInterceptor
-import software.amazon.awssdk.services.s3.handlers.EndpointAddressInterceptor
-import software.amazon.awssdk.services.s3.handlers.PutObjectInterceptor
+import software.amazon.awssdk.services.s3.internal.handlers.CreateBucketInterceptor
+import software.amazon.awssdk.services.s3.internal.handlers.DecodeUrlEncodedResponseInterceptor
+import software.amazon.awssdk.services.s3.internal.handlers.DisableDoubleUrlEncodingInterceptor
```

Comment:
Do we even need these anymore?

Reply:
Yes, seems Sam didn't get the codegen'ed change in time according to Matt